showpath option added to finder so people can see a folder's full pat…

bressan3 opened the pull request:
showpath option added to finder so people can see a folder's full path on the top bar of the opened Finder window instead of just seeing the folder's name:

m finder showpath YES:

![screen shot 2017-05-12 at 1 49 20 am](https://cloud.githubusercontent.com/assets/13811157/25982755/8e74280a-36b5-11e7-8ad9-4b44e7c55892.png)


m finder showpath NO:

![screen shot 2017-05-12 at 1 49 35 am](https://cloud.githubusercontent.com/assets/13811157/25982730/654e5fb8-36b5-11e7-9c71-85a0db94b2cf.png)

dcordero commented on plugins/finder:
```diff
@@ -112,6 +115,36 @@ desktop(){
+            defaults write com.apple.finder _FXShowPosixPathInTitle -bool false
+            ;;
+        *)
+            EXTENSION_STATUS=$(defaults read NSGlobalDomain _FXShowPosixPathInTitle 2>/dev/null)
```
I am getting an error here in macOS Sierra Version 10.12.4 (16E195)

```
$ m-cli finder showpath
We can't read _FXShowPosixPathInTitle property
```

I got it working applying the following patch:
```
- EXTENSION_STATUS=$(defaults read NSGlobalDomain _FXShowPosixPathInTitle 2>/dev/null)
+ EXTENSION_STATUS=$(defaults read com.apple.finder _FXShowPosixPathInTitle 2>/dev/null)
```

bressan3 commented:
Just fixed it by changing the file name to **com.apple.finder** like you did. I also added the  ... m finder [... | **showpath** | ...] option in both finder and Readme.md files for user reference.

rgcr commented:
Thank you so much for this 👍
